model_compression_toolkit/qat/keras/quantization_facade.py

```diff
@@ -148,16 +152,114 @@ def keras_quantization_aware_training_init(in_model: Model,
 
         tg = ptq_runner(tg, fw_info, fw_impl, tb_w)
 
-        qat_model, user_info = model_builder(tg, fw_info=fw_info)
+        qat_model, user_info = model_builder(tg, fw_info=fw_info, fw_impl=fw_impl)
 
         user_info.mixed_precision_cfg = bit_widths_config
 
         return qat_model, user_info, QUANTIZATION_CONFIGS_DICT
 
+
+    def keras_quantization_aware_training_export(in_model: Model):
```

Comment:
Rename the function from export to finalizing or something alone the line( this is true in several locations).